src/model/asset.ts

```diff
@@ -0,0 +1,38 @@
+import mongoose, { Schema, Document } from 'mongoose';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * @exports
+ * @interface IDocumentSession
+ * @extends {Document}
+ */
+export interface IDocumentAsset extends Document {
+  uuid: String;
+  articleId: Schema.Types.ObjectId;
+  s3link: String;
+  type: String;
+  uri: String;
+  created: Date;
+  updated: Date;
+}
+
+export interface IAsset {
+  uuid: String;
+  articleId: Schema.Types.ObjectId;
+  s3link: String;
+  type: String;
+  uri: String;
+  created: Date;
+}
+
+export const SchemaAsset = new Schema({
+  uuid: { type: String, default: uuidv4, unique: true, index: true },
+  articleId: { type: Schema.Types.ObjectId, ref: 'Article' },
```

Comment:
Do we need `author` here? I think `author` -> `user` is make sense.

Reply:
okay, I will change author field to user.